pc/device.c

```diff
@@ -31,9 +31,17 @@ struct ResidentKeyStore {
     CTAP_residentKey rks[RK_NUM];
 } RK_STORE;
 
+
```

Comment:
I'd remove these comments.  Could also remove forward declarations by moving the functions to be above where they are called.